video/out/hwdec/hwdec_drmprime.c

```diff
@@ -116,6 +116,7 @@ static int init(struct ra_hwdec *hw)
      */
     int num_formats = 0;
     MP_TARRAY_APPEND(p, p->formats, num_formats, IMGFMT_NV12);
+    MP_TARRAY_APPEND(p, p->formats, num_formats, IMGFMT_AVPIXFMT_START + AV_PIX_FMT_NV16);
```

Comment:
should be `pixfmt2imgfmt(AV_PIX_FMT_NV16)`

Reply:
thanks, i was looking for a macro or utility function for this

Reply:
should i squash to a new single commit?

Reply:
Please squash. Also pedantically, put it as the third item in the list, not the second. We would order 4:2:0 formats ahead of 4:2:2

Reply:
done with lowercase title and subsystem as well

Reply:
Thanks. Do fix whatever the test build is complaining about. Looks like a missing include.